stdlib/test/utils/test_stringref.mojo

```diff
@@ -108,47 +108,6 @@ def test_find():
     assert_equal(StringRef("").find("abc"), -1)
 
 
-fn test_stringref_split() raises:
```

Comment:
**Suggestion** Can you please port this test to `test_string_slice.mojo` before we merge this? We don't have any tests it looks like for `StringSlice.strip.`

Reply:
I wanted to add them in PR  #3887 but `StringSlice` doesn't yet have the method so I left the tests to be merged later on in PR #3528